lib/tools.js

```diff
@@ -193,6 +193,15 @@ function prettifierMetaWrapper (pretty, dest) {
 
 function createArgsNormalizer (defaultOptions) {
   return function normalizeArgs (opts = {}, stream) {
+    // support stream as a string
+    if (typeof opts === 'string') {
```

Comment:
this could be one block with an ||

Reply:
It can't, or I would need to add one if inside, making it harder to read. I need to pass either opts or stream to `new SonicBoom`.